feedback/static/html/feedback_instructor.html

```diff
@@ -0,0 +1,50 @@
+{% load i18n %}
+
+<div class="feedback-instructor-wrapper"></div>
+<div class="paragon-styles">
+    <div class="pgn__data-table-container">
+      <table role="table" class="pgn__data-table is-striped">
+        <thead>
+          <tr role="row">
+            <th colspan="1" role="columnheader">
+              <span class="d-flex align-items-center"><span>{% trans "Unit" %}</span></span>
+            </th>
+            <th colspan="1" role="columnheader">
+              <span class="d-flex align-items-center"><span>{% trans "Number of ratings" %}</span></span>
+            </th>
+            <th colspan="1" role="columnheader">
+              <span class="d-flex align-items-center"><span>{% trans "Average rating" %}</span></span>
+            </th>
+            <th colspan="1" role="columnheader">
+              <span class="d-flex align-items-center"><span>{% trans "Feedback " %}</span></span>
+            </th>
+          </tr>
+        </thead>
+        <tbody role="rowgroup">
+          {% for block in blocks %}
+          <tr role="row" class="pgn__data-table-row">
+            <td role="cell" class="pgn__data-table-cell-wrap">
+              <a class="admin-panel" href="{{ block.url }}"><button class="btn btn-primary" ="button">{% trans "See Unit" %}: {{ block.parent }}</button></a>
+            </td>
+            <td role="cell" class="pgn__data-table-cell-wrap">
+              <ul>
+                {% for vote_aggregate in block.vote_aggregate %}
+                <li>{{ vote_aggregate.scale_text }}: {{ vote_aggregate.count }}</li>
+                {% endfor %}
+              </ul>
+            </td>
+            <td role="cell" class="pgn__data-table-cell-wrap">{{ block.average_rating }}</td>
+            <td role="cell" class="pgn__data-table-cell-wrap">
+              {% for feedback in block.answers %}
+                <div class="feedback-block-answer"> * {{ feedback.user_freeform }}. Vote: {{ feedback.user_vote }}</div>
```

Comment:
What do you think about this change?
```html
<ul>
  {% for feedback in block.answers %}
  <li>{% trans "Comment" %}: {% if feedback.user_freeform %}{{ feedback.user_freeform }}{% else %}{% trans "No comment" %}{% endif %}. {% trans "Vote" %}: {{ feedback.user_vote }}</li>
  {% endfor %}
</ul>
``` 
Example:

![image](https://github.com/openedx/FeedbackXBlock/assets/64033729/93dff4dd-74b5-45d9-8051-aec60e13b3cd)

Reply:
I've hidden answers without feedback as that's the whole purpose of that column

Reply:
![image](https://github.com/openedx/FeedbackXBlock/assets/33465240/1347ca23-6e51-41da-bac2-d19b789c1de7)

This makes more sense I think